src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisWorkerService.java

```diff
@@ -260,7 +264,7 @@ public Consumer<Message<String>> consumeRun() {
                 }
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
-            } catch (Exception e) {
+            } catch (Exception | OutOfMemoryError e) {
```

Comment:
I don't think we should catch OutOfMemorryError

Reply:
It's the only easy way i found to be sure to send the failure notification to study-server, to set the computation status in the database, and then to have the front properly refreshed ...